### PR TITLE
scrolling

### PR DESCRIPTION
**Pull request by pipermerriam**
!!!!!

wizard

### CODE REVIEW

File: arst.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
```
Comment (wlonk) [inline arst.py:1]: Can you maybe integrate this `arst` stuff with the existing example script?

File: arst.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    ls_box._text_offset -= 1
+    app.root.dirty = True
+
+
+#@app.on(curses.KEY_DOWN)
+@app.on('k')
+def scroll_down(app):
+    app.log('in keydown')
+    app.log('offset is {0}'.format(ls_box._text_offset))
+    ls_box._text_offset += 1
```
Comment (wlonk) [inline arst.py:72]: Adding a `scroll` method or something to `Box` is probably better; it can handle the offset and the dirtying, perhaps?

**Comment by wlonk** [comment]
If you mention #21 in the commit message, Waffle and GitHub will pick up on it.

**Comment by pipermerriam** [comment]
Chances are you'll have to take this and run with it.  I'm not really very available to take this anywhere useful unless you pin me down and work on it with me somehow.

**Pull request closed without merging**